baseline/OSPS-QA.yaml

```
@@ -237,3 +227,65 @@ criteria:
     control_mappings:       
       BPB: B-G-3
     security_insights_value: # TODO
+
+  - id: OSPS-QA-11
+    maturity_level: 2
+    criterion: |
+      All released software assets MUST be
+      delivered with a software bill of materials
+      (SBOM) in SPDX or CycloneDX format accounting
+      all direct software dependencies with their
+      associated version identifiers.
+    rationale: |
+      Provide transparency and accountability for
+      the project's dependencies in a standard format
+      allowing automated systems to understand the 
+      software's dependencies and versions.
+    details: |
+      It is recommended to use a CycloneDX or SPDX
+      file that is auto-generated at build time by
+      a tool that has been vetted for accuracy.
+      This enables users to ingest this data in a
+      standardized approach alongside other
+      projects in their environment.
+    control_mappings: 
+      BPB: Q-S-9
+      CRA: 13.24, I.II.1, VII.2
+      SSDF: PS3, PW4
+      CSF: ID.AM-02
+      OC: 4.3.1
+      OCRE: 486-813, 124-564, 863-521
+    security_insights_value: # TODO
+
+  - id: OSPS-QA-12
+    maturity_level: 3
+    criterion: |
+      The project will provide both source and build software bills 
+      of materials the following data elements for the components 
+      used in the build: 
```

```suggestion
      included in the build: 
```

Few should care about the platform used to run a build. Knowing the tools used to generate & test are of interest, but let's walk before we run.